### PR TITLE
Introduce DistributedStorageAdmin that is an administrative interface…

### PR DESCRIPTION
**Pull request by brfrn169**
… for distributed storage implementations

I added DistributedStorageAdmin that is an administrative interface for distributed storage implementations. createTable/dropTable/truncateTable have not been implemented yet for all the adapters. We can implement them in another PR.

Please take a look!

### CODE REVIEW

File: src/main/java/com/scalar/db/service/AdminModule.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package com.scalar.db.service;
```
Comment (feeblefakie) [inline src/main/java/com/scalar/db/service/AdminModule.java:1]: I'm wondering if we need another module for Admin that is identical to StorageModule.
Is there any problem updating StorageModule just by adding the line?
```
bind(DistributedStorageAdmin.class).to(config.getAdminClass()).in(Singleton.class);
```

**Review by feeblefakie — COMMENTED** [review]
Overall looking good! Nice improvement!
I left one question.

File: src/main/java/com/scalar/db/service/AdminModule.java (hunk 1 above)
Comment (brfrn169) [reply src/main/java/com/scalar/db/service/AdminModule.java:1]: Thank you for reviewing this!

> I'm wondering if we need another module for Admin that is identical to StorageModule.

Yes, it's better!  I will fix it in that way. Thanks.

**Review by feeblefakie — APPROVED** [review]
LGTM!

**Comment by feeblefakie** [comment]
@yito88 PTAL when you get a chance!

**Pull request merged**